packages/sdk/src/browser-wallets/xverse/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { BitcoinNetworkType } from "sats-connect";
+
+import { BrowserWalletNetwork } from "../../config/types";
+
+export function fromXOnlyToFullPubkey(xOnly: string): string {
+  return `03${xOnly}`; // prepend y-coord/tie-breaker to x-only
```

Comment:
Technically it's the y-coord of the elliptic curve. Xverse returns just the X-Only part of the public key. This can actually be `02` or `03` but I think we have always used `03` and it hasn't been an issue. 
@iamcrazycoder please correct me if I'm wrong.

Reply:
Refer to this slack convo for more info: https://cakedefi.slack.com/archives/C060ZPVNAS3/p1698647423012619?thread_ts=1698645355.924329&cid=C060ZPVNAS3

Reply:
could you add this explanation in the code comment? (e.g. `03` is the y-coord/tie-breaker of the elliptic curve, etc.) that way for posterity, PR maintainers won't be required to read PR notes to know what y-coord is

Reply:
So I did a deep dive in to the issue, and apparently this is wrong. This should be `02` instead of `03`, as according to [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) the y-coordinate should be even. This hasn't been an issue because whenever the pubkey needs to be used we call the `toXOnly` function that strips away the first byte. But this is technically wrong and may result in errors down the line in future.

Will fix.

CC: @iamcrazycoder

Reply:
Fixed